manticore/platforms/linux.py

```diff
@@ -1,18 +1,20 @@
 import fcntl
 
-import cgcrandom
-import weakref
+import logging
+import random
 import errno
-import os, struct
+import os
+import struct
+
+import cgcrandom
```

Comment:
Since we're reformatting, should probably use relative project import for `cgcrandom`